vignettes/setup/setup.R

```diff
@@ -1,6 +1,12 @@
+are_suggested_pkgs_needed_for_vignettes_available <- function() {
+  pkgs_needed_for_vignettes <- c("tidyverse")
+  all(purrr::map_lgl(pkgs_needed_for_vignettes, rlang::is_installed))
+}
```

Comment:
I know this whole function can be simplified just by using `rlang::is_installed("tidyverse")`, but I am trying to be future-proof here. If `{dm}` vignettes need any additional deps in the future, we can simply add those to the `pkgs_needed_for_vignettes` vector.